libs/langgraph/src/pregel/call.ts

```diff
@@ -0,0 +1,63 @@
+import { RunnableConfig, RunnableSequence } from "@langchain/core/runnables";
+import { AsyncLocalStorageProviderSingleton } from "@langchain/core/singletons";
+import { CONFIG_KEY_CALL, RETURN, TAG_HIDDEN } from "../constants.js";
+import { ChannelWrite, PASSTHROUGH } from "./write.js";
+import { RetryPolicy } from "./utils/index.js";
+import { Promisified } from "./types.js";
+import { RunnableCallable, type RunnableCallableArgs } from "../utils.js";
+/**
+ * Cache for storing runnable sequences to avoid recreating them
```

Comment:
What does this apply to?

Reply:
@benjamincburns didn't we decide not to do this?

Reply:
I apparently missed the comment.